qutebrowser/config/configdata.yml

```diff
@@ -441,17 +442,17 @@ content.javascript.can_close_tabs:
   default: false
   type: Bool
   backend: QtWebKit
-  desc: Whether JavaScript can close tabs.
+  desc: Enable JavaScript to close tabs.
 
 content.javascript.can_open_tabs_automatically:
   default: false
   type: Bool
-  desc: Whether JavaScript can open new tabs without user interaction.
+  desc: Enable JavaScript to open new tabs without user interaction.
```

Comment:
Maybe those should be "Allow JavaScript to ..." instead of "Enable JavaScript to ..."?